### ISSUE TITLE
Include more error information from Orbit API

### ISSUE THREAD

**Issue by jpoehnelt** [orbit-love/community-js-youtube-orbit#2]
The following error might be empty. 

https://github.com/orbit-love/community-js-youtube-orbit/blob/928f6800e47619f20bf5538657e88b7ec3fc9e38/src/orbit.js#L27

As seen here:

[![Screenshot 2021-05-20 6 55 50 PM](https://user-images.githubusercontent.com/3392975/119066450-07651080-b99d-11eb-9c65-fd09f2ec1692.png)](https://github.com/googlemaps/.github/runs/2635242012?check_suite_focus=true)

**Comment by jpoehnelt** [comment]
Apparently this was caused by an invalid channel id.

**Comment by phazonoverload** [comment]
Error handling and input validation across all the JS community integration will be a theme of the next few weeks. Right now these should work reasonably well, but I will be spending some time to shore them all up so if they don't work it should be clear why. 😄

**Comment by lizrice** [comment]
Probably falls into the general bucket of error handling that @phazonoverload mentions (but let me know if you'd rather I raised this as a separate issue). We hit an error because we have comments turned off on (at least one) video. I'm not sure whether this stopped the process of gathering any comments at all?

```
Run npx @orbit-love/youtube --comments --hours=24
npx: installed 24 in 3.031s
(node:1494) UnhandledPromiseRejectionWarning: Error: Error: The video identified by the <code><a href="/youtube/v3/docs/commentThreads/list#videoId">videoId</a></code> parameter has disabled comments.
    at /home/runner/.npm/_npx/1494/lib/node_modules/@orbit-love/youtube/src/comments.js:22:67
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async /home/runner/.npm/_npx/1494/lib/node_modules/@orbit-love/youtube/src/comments.js:21:34
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1494) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 12)
(node:1494) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by phazonoverload** [comment]
Hey @lizrice - this is a different issue not related to the Orbit API but instead the YouTube one. I’ve opened a new issue for it 😊

**Comment by phazonoverload** [comment]
Just released 1.0.0 which should address this meaningfully.
